src/bench/bench.cpp

```diff
@@ -41,15 +41,37 @@ void GenerateTemplateResults(const std::vector<ankerl::nanobench::Result>& bench
 
 } // namespace
 
+std::map<std::string, benchmark::PriorityLevel> map_priority_level = {
+    std::make_pair("high", benchmark::PriorityLevel::HIGH),
+    std::make_pair("medium", benchmark::PriorityLevel::MEDIUM),
+    std::make_pair("low", benchmark::PriorityLevel::LOW),
+    std::make_pair("all", benchmark::PriorityLevel::ALL)
+};
+
+uint8_t benchmark::ToPriorityLevel(const std::string& str)
+{
+    auto it = map_priority_level.find(str);
+    if (it == map_priority_level.end()) throw std::runtime_error(strprintf("Unknown priority level %s", str));
+    return static_cast<uint8_t>(it->second);
+}
+
+std::string benchmark::PriorityToString(PriorityLevel level)
+{
+    for (const auto& it : map_priority_level) {
+        if (it.second == level) return it.first;
+    }
+    throw std::runtime_error(strprintf("Unknown priority level %d", level));
```

Comment:
I'm not sure why this compiles because I think you can't pass a `PriorityLevel` like that:

```suggestion
    throw std::runtime_error(strprintf("Unknown priority level %u", static_cast<uint8_t>(level)));
```

Also, I don't think this is a runtime error because it can only be hit when `map_priority_level` is out of sync with `PriorityLevel`?

Reply:
Better `PriorityToString(level)` instead. The number isn't so informative.